XMonad/Doc/Extending.hs

```diff
@@ -1111,6 +1111,11 @@ external utilities.
 
 A non complete list with a brief description:
 
+* "XMonad.Util.ClickableWorkspaces":
+    Provides clickablePP, which when applied to the PP pretty-printer used by
+    'dynamicLogWithPP' of 'XMonad.Hooks.DynamicLog', will make the workspace tags
```

Comment:
This should be `XMonad.Hooks.DynamicLog.dynamicLogWithPP''` and `"XMonad.Hooks.DynamicLog"`
See https://www.haskell.org/haddock/doc/html/ch03s08.html#idm140354810751008